middlewares/index.js

```diff
@@ -60,6 +61,21 @@ class Middleware {
       });
     }
   }
+
+  /**
+     * checks blacked listed tokens.
+     * @param {request} req .
+     * @param {response} res The second number.
+     * @param {next} next The second number.
+     * @returns {void} calls next on success.
+     */
+  static async generateReadTime(req, res, next) {
+    const { body } = req.body;
+
+    const { text } = readingTime(body);
+    req.body.readTime = text;
```

Comment:
spread the req.body object to be
``
req.body = { ..req.body, readTime: text }
``